syslog/src/g3log/syslogsink.hpp

```diff
@@ -57,6 +58,10 @@ class SyslogSink
 
     void setLevel(LogLevel level, int syslevel);
 
+    // Alternative methods releaving C-string argument lifetime constraints:
+    void setLogHeaderNoJn(g3::DltrStr change) { _header = change.c_str(); }
```

Comment:
I'd suggest placing each of them right under the original function. That way it's easier to see that there is an alternative function.  The comment is great. I'd have the comment above each function